src/main/java/emissary/pickup/PickUpPlace.java

```diff
@@ -552,10 +568,27 @@ protected boolean processDataObject(IBaseDataObject d, String fixedName, File th
         dataObjectCreated(d, theFile);
         logger.info("**Deploying an agent for {} and object {} forms={} simple={}", fixedName, d.getInternalId(), d.getAllCurrentForms(),
                 (simpleMode ? "simple" : ""));
+
+        // If object tracing log that agent is being deployed for fixedName (filename)
+        if (useObjectTraceLogger) {
+            objectTraceLog(d);
+        }
+
         assignToPooledAgent(d, -1L);
         return true;
     }
 
+    /**
+     * Creates an entry in the object trace log. Can be overridden if desired.
+     * 
+     * @param d The IBDO
+     */
+    public void objectTraceLog(IBaseDataObject d) {
+        objectTraceLogger.info(appendEntries(
+                ObjectTracing.createTraceMessageMap(new String[] {"PickUp", d.getFilename(), String.valueOf(System.currentTimeMillis())})).toString(),
```

Comment:
We know that every trace message is going to need the timestamp in this format. Does it make sense to just have the ObjectTracingUtil add that in?
I don't think we need to have the timestamp field configurable. Just always part of the message. 
Just a thought. Open for more discussion